news-app/src/components/Votes.js

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import {patchArticleByArticleID} from "../utils/api"
+
+export default function Vote({votes, articleId}) {
+    const [voteChange, setVoteChange] = useState(0)
+    const [voted, setVoted] = useState(false)
+
+
+function incVote() {
+    setVoted(false)
+    setVoteChange((currVoteChange) => currVoteChange + 1 )
+    patchArticleByArticleID(articleId, 1)
+    setVoted(true)
+    .catch((err) => {
+            setVoteChange((currVoteChange) => currVoteChange - 1)
+            console.error(err, "mitch error")
```

Comment:
What is this .catch tagged on to?
At the moment, when then api request fails, it still says 'thanks for voting' and the number still stays changed, which isnt reflective of the database